src/routers/middleware_router.rs

```diff
@@ -9,6 +9,8 @@ use matchit::Node;
 
 use anyhow::{bail, Error, Result};
 
+use super::router::RouteType;
```

Comment:
I don't think `super` is required anymore in Rust.

Reply:
another option is to write like this.
```
use crate::routers::types::MiddlewareRouteType;
```

Reply:
@suhailmalik07 , yep. That is the preferred way.

Reply:
update with this.